static/app/views/issueList/searchBar.tsx

```diff
@@ -93,6 +98,15 @@ const EXCLUDED_TAGS = ['environment'];
 function IssueListSearchBar({organization, tags, ...props}: Props) {
   const api = useApi();
   const {selection: pageFilters} = usePageFilters();
+  const {tags: issueTags} = useFetchIssueTags({
+    org: organization,
+    projectIds: pageFilters.projects.map(id => id.toString()),
+    keepPreviousData: true,
+    enabled: organization.features.includes('issue-stream-search-query-builder'),
+    start: pageFilters.datetime.start,
```

Comment:
Noticed a couple things here:

- Selecting an absolute date range doesn't work (need to use the `getUtcDateString()` function)
- Changing the time range seems to cause double requests, one with `start/end` as empty string and one with them undefined

Reply:
Thanks for catching this - should be fixed now!